vespa_analysis/vespa_msa.py

```diff
@@ -34,24 +34,32 @@ def cons_stats(ref_id,ref_aln,cons_class):
             c=c+1
             cons_vector.append(int(cons_class[c-1]))
     return cons_vector
+def arg_parser():
+    parser = argparse.ArgumentParser(description='Computes SAV and Conservation scores statistics')
+    parser.add_argument('--vespa_map_out', type=str, action='store', help='FULL Path of vespa mapped score output csv file',required=True)
+    parser.add_argument('--ref_id', type=str, action='store', help='reference sequence ID',required=True)
+    parser.add_argument('--ref_aln', type=str, action='store', help='MSA reference sequence',required=True)
+    parser.add_argument('--cons_class', type=str, action='store', help='Conservation classes of reference sequence',required=True)
+    parser.add_argument('--gen_id', type=str, action='store', help='generated sequence ID',required=True)
+    parser.add_argument('--gen_aln', type=str, action='store', help='MSA generated sequence',required=True)
+    args = parser.parse_args()
+    
+    mapped_scores = args.vespa_map_out
+    ref_id = args.ref_id
+    ref_aln = args.ref_aln
+    cons_class = args.cons_class
+    gen_id = args.gen_id
+    gen_aln = args.gen_aln
+    return [mapped_scores,ref_id,ref_aln,cons_class,gen_id,gen_aln]
 
-parser = argparse.ArgumentParser(description='Computes SAV and Conservation scores statistics')
-parser.add_argument('--vespa_map_out', type=str, action='store', help='FULL Path of vespa mapped score output csv file',required=True)
-parser.add_argument('--ref_id', type=str, action='store', help='reference sequence ID',required=True)
-parser.add_argument('--ref_aln', type=str, action='store', help='MSA reference sequence',required=True)
-parser.add_argument('--cons_class', type=str, action='store', help='Conservation classes of reference sequence',required=True)
-parser.add_argument('--gen_id', type=str, action='store', help='generated sequence ID',required=True)
-parser.add_argument('--gen_aln', type=str, action='store', help='MSA generated sequence',required=True)
-args = parser.parse_args()
-
-
-## Inputs ##
-mapped_scores = args.vespa_map_out
-ref_id = args.ref_id
-ref_aln = args.ref_aln
-cons_class = args.cons_class
-gen_id = args.gen_id
-gen_aln = args.gen_aln
+##parsing inputs:
+inputs = arg_parser()
```

Comment:
check my comment above regarding unpacking the outputs of a function.

Reply:
you can do:  `mapped_scores, ref_id, ref_aln, cons_class, gen_id, gen_aln = arg_parser()`